CHANGELOG.rst

```diff
@@ -5,6 +5,19 @@ redhat.satellite_operations Release Notes
 .. contents:: Topics
 
 
+v2.0.0
+======
+
+Minor Changes
+-------------
+
+- cloud_connector role - don't mark proxy.conf systemd drop-in word-inaccessible, there is no benefit and systemd warns about this (https://bugzilla.redhat.com/show_bug.cgi?id=2169682)
+
+Breaking Changes / Porting Guide
+--------------------------------
+
+- ansible_repositories role - removed as it's not required anymore since EL 8.6 contains ``ansible-core``
```

Comment:
This role was never part of *this* collection, so I'd omit the confusing changelog entry.

Interestingly this also "breaks" the versioning (this release is more a bugfix, maybe a feature with all the CI fixes, not major), but I'd keep it in sync nevertheless.

Reply:
Agreed.